app/assets/stylesheets/playtime.scss

```diff
@@ -71,6 +72,16 @@ h3 {
   float: right;
 }
 
+.alert {
+  margin: 10px;
+}
+
+#notice {
```

Comment:
In general, I'd suggest using ids as unique identifiers (for JS actions, etc.) and classes for styling unless there's a specific reason not to. Ids are unique, so styling them doesn't promote code sharing or those other nice DRY things. 

If an element is unique on a page, it doesn't *really* matter, but since developers expect the classes to contain styles, it's nice not to surprise them.